modules/assisted-installer-installing-the-cluster.adoc

```diff
@@ -0,0 +1,15 @@
+// This is included in the following assemblies:
+//
+// assisted-installer-installing.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-cluster_{context}"]
+= Installing the cluster
+
+Once you have completed the configuration and all the nodes are *Ready*, you can begin installation. The installation process takes a considerable amount of time, and you can monitor the installation from the {ai-full} web console. Nodes will reboot during the installation and they will initialize after installation.
```

Comment:
Do we want to specifically mention how to monitor?
- Progress bars
- Cluster events